scripts/trigger-maps-documentation-deploy-steps.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+function step { >&2 echo -e "\033[1m\033[36m* $@\033[0m"; }
+function finish { >&2 echo -en "\033[0m"; }
+trap finish EXIT
+
+SDK_FLAVOR=${SDK_FLAVOR:-"maps"}
```

Comment:
Are there scenarios in which we'll have an existing `SDK_FLAVOR` env variable or will it always just be `"maps"`?

Reply:
Will always be `"maps"`